src/ethereum-identity-provider.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+const IdentityProvider = require('./identity-provider')
+const { Wallet, utils } = require('ethers')
+const type = 'ethereum'
+
+class EthIdentityProvider extends IdentityProvider {
+  constructor(options) {
```

Comment:
We can probably remove the constructor as it's not doing anything other than calling the parent constructor?